.golangci.yml

```diff
@@ -143,11 +143,16 @@ linters-settings:
           desc: "replaced by internal packages like pkg/util/ptr"
         - pkg: github.com/onsi/ginkgo$
           desc: "replaced by github.com/onsi/ginkgo/v2"
-
+  errorlint:
+    # Check whether fmt.Errorf uses the %w verb for formatting errors.
```

Comment:
I _want_ `Errorf` to use `%w` to format errors. Why would we wish to disable that?

Reply:
That disables the lint for not using %w. There's lots of places where we're not and something that can be addressed later (along with the other disabled check)